### DISCUSSION TITLE
NSFW

### DISCUSSION THREAD

**Discussion by sirPhoebus** [hacksider/Deep-Live-Cam#33] [General]
What does this toggle do exactly ?
It prevents censured or something like that ...But how ?

**Comment by hacksider** [comment]
It will just allow you to render anything NSFW.

**Comment by 123LiVo321** [comment]
the nsfw switch is gone... that means...?

**Reply by hacksider** [reply]
I hide it

**Reply by 123LiVo321** [reply]
....because...?  is there a way to re-hide it? :P

**Reply by hacksider** [reply]
because I can, hahaha! You can just check the code to rehide it (it's just commented)

**Reply by 123LiVo321** [reply]
i did, eventhough i didnt found the commented section [edit: oh, now I do!], I figure out enable the mode in core.py ....just curious why you did it... nvm

**Reply by Buop32545** [reply]
I found the commented code and rehide it but it always said porcess ignored can someone help me? sry im new to this

**Reply by hacksider** [reply]
We can't answer that since we already removed (hide) that feature 😅

You can just figure it out on your own 😝
